code/game/objects/items/weapons/grenades/flashbang.dm

```diff
@@ -136,7 +156,21 @@
 
 /obj/item/grenade/flashbang/nt
 	name = "NT FBG \"Holy Light\""
-	desc = "An old \"NanoTrasen\" flashbang grenade, modified to spread the light of god."
+	desc = "An old \"NanoTrasen\" flashbang grenade. \
+			There's an inscription along the bands. \'Rome will glow with the light of her Lord.\'"
 	icon_state = "flashbang_nt"
 	item_state = "flashbang_nt"
 	matter = list(MATERIAL_BIOMATTER = 15)
+
+/obj/item/grenade/flashbang/nt/flashbang_without_the_bang(turf/T, mob/living/carbon/M)
+	if(M.get_core_implant(/obj/item/implant/core_implant/cruciform))
+		to_chat(M, span_singing("You are blinded by the light of the Angels!"))
+		if (M.HUDtech.Find("flash"))
+			flick("e_flash", M.HUDtech["flash"])
+		return
+	..()
+
+/obj/item/grenade/flashbang/nt/flashbang_bang(var/turf/T, var/mob/living/carbon/M, var/explosion_text = "BANG", var/stat_reduction = TRUE, var/intensity = FALSE)
+	if(M.get_core_implant(/obj/item/implant/core_implant/cruciform))
+		intensity += 1
```

Comment:
You give intensity a default value as if it was a bool `var/intensity = FALSE` but you keep using it as an int (`intensity += 1` and `bang_intensity += intensity`). It's ok but a bit confusing.